### PR TITLE
fix: update amend pegged order margin expected behaviour

### PR DESCRIPTION
**Pull request by Sohill-Patel**
updated expected behaviour of a pegged order amend
remove duplicate ac

### CODE REVIEW

File: protocol/0019-MCAL-margin_calculator.md [hunk 1]
```diff
@@ -163,13 +163,11 @@ There should be an additional amount `limit price x size x margin factor = 15910
 
 - When the party increases the pegged sell order offset price, the order margin should be reduced (<a name="0019-MCAL-057" href="#0019-MCAL-057">0019-MCAL-057</a>)
 
-- When the party increases the pegged order size and the party's general account does not contain sufficient funds to cover any increases to the order margin account to be equal to side margin then the order should be stopped (<a name="0019-MCAL-052" href="#0019-MCAL-052">0019-MCAL-052</a>)
+- When the party increases the pegged order size and the party's general account does not contain sufficient funds to cover any increases to the order margin account to be equal to side margin then the amendment is rejected and the original order is not effected (<a name="0019-MCAL-052" href="#0019-MCAL-052">0019-MCAL-052</a>)
```
Comment (TomMcL) [inline protocol/0019-MCAL-margin_calculator.md]:
```suggestion
- When the party increases the pegged order size and the party's general account does not contain sufficient funds to cover any increases to the order margin account to be equal to side margin then the amendment is rejected and the original order is not affected (<a name="0019-MCAL-052" href="#0019-MCAL-052">0019-MCAL-052</a>)
```

File: protocol/0019-MCAL-margin_calculator.md [hunk 2]
```diff
@@ -163,13 +163,11 @@ There should be an additional amount `limit price x size x margin factor = 15910
...
 
-- When the party increases the pegged order price and the party's general account does not contain sufficient funds to cover any increases to the order margin account to be equal to side margin then the order should be stopped (<a name="0019-MCAL-075" href="#0019-MCAL-075">0019-MCAL-075</a>)
+- When the party increases the pegged order price and the party's general account does not contain sufficient funds to cover any increases to the order margin account to be equal to side margin then the amendment is rejected and the original order is not effected (<a name="0019-MCAL-075" href="#0019-MCAL-075">0019-MCAL-075</a>)
```
Comment (TomMcL) [inline protocol/0019-MCAL-margin_calculator.md]:
```suggestion
- When the party increases the pegged order price and the party's general account does not contain sufficient funds to cover any increases to the order margin account to be equal to side margin then the amendment is rejected and the original order is not affected (<a name="0019-MCAL-075" href="#0019-MCAL-075">0019-MCAL-075</a>)
```

**Review by TomMcL — CHANGES_REQUESTED** [review]
I think the deleted AC will need removing from features.json too

**Comment by Sohill-Patel** [comment]
> I think the deleted AC will need removing from features.json too

@TomMcL i did check and ac `0019-MCAL-077` is not in there

**Review by TomMcL — APPROVED** [review]

**Pull request merged**